tests/101_read_seq_sync_bigfile.fio

```diff
@@ -0,0 +1,8 @@
+; Reads a 4G file in sequential order with 512K block size synchronously. O_DIRECT is turned off in this test.
+; This should work similar to what dd does. (https://github.com/coreutils/coreutils/blob/master/src/dd.c)
+[seq-reader]
+ioengine=sync
+rw=read
+bs=512
+direct=0
+size=4G
```

Comment:
Just to have some file size that is big enough to get enough data for aggregation and keep tests running long enough. I will add something about this in the README.